content/resources/operator/_index.md

```diff
@@ -378,3 +378,15 @@ resources:
 	- my_program-TEST1_name3_downloader
 	- my_program-TEST1_name4_downloader
 ```
+
+## 8. ETL and Data Explorer Configurations
+
+ETL stands for “extract, transform, load”. In Gen3, it refers to the process of moving data from the PostgreSQL database (graph model) to the ElasticSearch database (flat model), which is carried out by the Gen3 Framework Service [Tube](https://github.com/uc-cdis/tube).
```

Comment:
I'm not sure if Tube is one of Gen3 Framework Service (G3FS). I think only a couple of serives (fence, arborist, indexd, sheepdog, etc.) are G3FS

I would just say "Gen3 service Tube"